packages/workflow/src/records/handler/archive.test.ts

```diff
@@ -55,6 +57,31 @@ describe('archive record endpoint', () => {
       )
     )
 
+    // mock fetching the practitioner's location hierarchy, derived from regLastOffice
+    mswServer.use(
+      rest.get(
+        'http://localhost:2021/locations/ce73938d-a188-4a78-9d19-35dfd4ca6957/hierarchy',
+        (_, res, ctx) => {
+          return res(
+            ctx.json([
+              fixtures.savedAdministrativeLocation({
+                id: '0f7684aa-8c65-4901-8318-bf1e22c247cb' as UUID,
+                name: 'Ibombo',
+                partOf: { reference: 'Location/0' }
+              }),
+              fixtures.savedAdministrativeLocation({
+                id: 'ce73938d-a188-4a78-9d19-35dfd4ca6957' as UUID,
+                name: 'Ibombo District Office',
+                partOf: {
+                  reference: 'Location/0f7684aa-8c65-4901-8318-bf1e22c247cb'
+                }
+              })
+            ])
+          )
+        }
+      )
+    )
```

Comment:
As I had mentioned in the other PR, we can move it here: https://github.com/opencrvs/opencrvs-core/blob/develop/packages/workflow/test/handlers.ts#L40-L57 as this is basically replacement for the previous practitioner location fetches. As it's common for most (if not all) the event transitions